ZConfig/cmdline.py

```diff
@@ -155,7 +157,7 @@ def get_section_info(self, type, name):
             bk = self.basic_key(s, pos)
             if name and self._normalize_case(s) == name:
                 L.append((optpath[1:], val, pos))
-            elif bk == type:
+            elif bk == type: # pragma: no cover
```

Comment:
There would be a *lot* or renames to do throughout the codebase. `type` (among others) is commonly shadowed; I think this code predates new-style classes and `type` as a builtin.

I'd be happy to see that done, but I believe it belongs in another PR.